shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2187,6 +2199,16 @@ - (void)triggerAutofillSave:(BOOL)saveEntries {
   [self addToInputParentViewIfNeeded:_activeView];
 }
 
+- (void)setPlatformViewTextInputClient:(long)platformViewID {
+  // No need to track the platformViewID for now (unlike in Android), because in iOS there can
+  // only be one single first responder. When a platform view becomes first responder, hide
+  // this dummy text input view (`_activeView`) for the previously focused widget.
+  [self removeEnableFlutterTextInputViewAccessibilityTimer];
+  _activeView.accessibilityEnabled = NO;
+  [_activeView removeFromSuperview];
+  [_inputHider removeFromSuperview];
+}
+
```

Comment:
This looks fine to me.

cc @justinmc, do you mind take a quick look at this to see if it makes sense?

Reply:
im gonna land it first since i've tested it many times, and it's not used just yet (until the framework part is complete), but let me know if any thing comes up